### PR TITLE
Move some articles from wrong sections

### PR DESCRIPTION
**Pull request by wouterj**
Finishes #5301 

| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets | #4416 |

### CODE REVIEW

File: cookbook/map.rst.inc [hunk 1]
```diff
@@ -34,11 +34,11 @@
   * :doc:`/cookbook/configuration/using_parameters_in_dic`
   * :doc:`/cookbook/configuration/front_controllers_and_kernel`
   * :doc:`/cookbook/configuration/external_parameters`
-  * :doc:`/cookbook/configuration/pdo_session_storage`
   * :doc:`/cookbook/configuration/apache_router`
   * :doc:`/cookbook/configuration/web_server_configuration`
   * :doc:`/cookbook/configuration/configuration_organization`
-  * :doc:`/cookbook/configuration/mongodb_session_storage`
+  * (configuration) :doc:`/cookbook/doctrine/pdo_session_storage`
+  * (configuration) :doc:`/cookbook/doctrine/mongodb_session_storage`
```
Comment (xabbuh) [inline cookbook/map.rst.inc]: (Doctrine)

File: cookbook/map.rst.inc [hunk 2]
```diff
@@ -189,8 +190,8 @@
   * :doc:`/cookbook/session/locale_sticky_session`
   * :doc:`/cookbook/session/sessions_directory`
   * :doc:`/cookbook/session/php_bridge`
-  * (configuration) :doc:`/cookbook/configuration/pdo_session_storage`
-  * (configuration) :doc:`/cookbook/configuration/mongodb_session_storage`
+  * (doctrine) :doc:`/cookbook/doctrine/pdo_session_storage`
```
Comment (javiereguiluz) [inline cookbook/map.rst.inc]: `doctrine` -> `Doctrine` ?

**Comment by WouterJ** [comment]
Thanks @javiereguiluz and @xabbuh

**Comment by javiereguiluz** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thanks Wouter.

File: cookbook/doctrine/index.rst [hunk 3]
```diff
@@ -14,4 +14,6 @@ Doctrine
     resolve_target_entity
     mapping_model_classes
     registration_form
+    pdo_session_storage
+    mongodb_session_storage
```
Comment (stof) [inline cookbook/doctrine/index.rst]: this look weird to me. These are not about Doctrine but about native PDO and MongoDB

Comment (stof) [inline cookbook/doctrine/index.rst]: the right location would be in ``/cookbook/session`` IMO

**Pull request merged**